hail/python/hailtop/utils/utils.py

```diff
@@ -410,16 +410,16 @@ def retry_response_returning_functions(fun, *args, **kwargs):
         errors += 1
         if errors % 10 == 0:
             log.warning(f'encountered {errors} bad status codes, most recent '
-                        f'one was {response.status_code}', exc_info=True)
+                        f'one was {response.status_code}')
```

Comment:
there's no exception here so python prints:
```
NoneType: None
```
removing exc_info removes this message.